### PR TITLE
Tyler/add parsed sources

### PR DESCRIPTION
**Pull request by tjdammann**
Added Google parsed sources, regenerated SDK based on changes to Swagger (e.g., removing some endpoints from Swagger), and updated auth tools to include correct URL

### CODE REVIEW

File: bruinen/src/bruinen/langchain/parsers/sql_parser.py [hunk 1]
```diff
@@ -97,13 +78,9 @@ def map_sql_types(type_var):
...
-            input_class_name.lower(), metadata, autoload=True, autoload_with=engine
-        )
+        sql_table = Table(input_class_name.lower(), metadata, autoload=True, autoload_with=engine)
 
-        template_table = "\nOnly use the following tables:\n" + str(
-            CreateTable(sql_table).compile(engine)
-        )
+        template_table = "\nOnly use the following tables:\n" + str(CreateTable(sql_table).compile(engine))
         # If the table isn't too large, would be better to also include example data
         template_start = """Given an input question, first create a syntactically correct {dialect} query to run, then look at the results of the query and return the answer. Use the following format:
```
Comment (tevonsb) [inline bruinen/src/bruinen/langchain/parsers/sql_parser.py:85]: Lets experiment with different prompts to see if we can get even better results?

Lets make a backlog task for this (low priority), fun thing to work on in any downtime

File: bruinen/src/bruinen/langchain/parsers/sql_parser.py [hunk 2]
```diff
@@ -97,13 +78,9 @@ def map_sql_types(type_var):
...
-        sql_table = Table(
-            input_class_name.lower(), metadata, autoload=True, autoload_with=engine
-        )
+        sql_table = Table(input_class_name.lower(), metadata, autoload=True, autoload_with=engine)
 
-        template_table = "\nOnly use the following tables:\n" + str(
-            CreateTable(sql_table).compile(engine)
-        )
+        template_table = "\nOnly use the following tables:\n" + str(CreateTable(sql_table).compile(engine))
         # If the table isn't too large, would be better to also include example data
```
Comment (tevonsb) [inline bruinen/src/bruinen/langchain/parsers/sql_parser.py:84]: An example row is certainly something we should consider.

Lets start a feature list and design doc for this, lets find the short-comings and improve on them

File: langchain_tools_example.py [hunk 3]
```diff
@@ -27,9 +38,7 @@ def parse_drafts(output, query):
     return output
 google_drafts_tool = GoogleGetDraftsTool(client=client, user_id=user_id, llm=input_llm, parse_output=parse_drafts)
 agent = initialize_agent([google_drafts_tool], agent_llm, agent='chat-zero-shot-react-description', verbose=True)
-result = agent.run("What are my Google drafts?")
-
-exit()
+result = agent.run("What are my Google drafts with q=null and pageToken='1'?")
```
Comment (tevonsb) [inline langchain_tools_example.py:41]: We've tested without explicit q values right?

**Review by tevonsb — APPROVED** [review]
Awesome!!

File: langchain_tools_example.py (hunk 3 above)
Comment (tjdammann) [reply langchain_tools_example.py:41]: Yep, haven't fully tested but have tested with q='to:tevon@bruinen.co' and it works great

**Pull request merged**